rb/Steepfile

```diff
@@ -1,8 +1,76 @@
-# frozen_string_literal: true
-
 target :lib do
-  signature 'sig' # Signature directory
+  signature 'sig', '.gem_rbs_collection/rubyzip' # Signature directory
```

Comment:
I don't see rubyzip committed, is this by mistake?

Reply:
@p0deje Without specifying rubyzip like this, we get the following error, because we have ruby zip as a dependency

So in our rbs_collection.yaml, when we run rbs collection install we get all the signatures we need

Here is the reference: https://github.com/ruby/gem_rbs_collection

<img width="1644" alt="Screenshot 2024-03-13 at 21 42 04" src="https://github.com/SeleniumHQ/selenium/assets/33221555/8b9fb66e-d38d-4267-92d2-6f2c0c82b02d">

<img width="516" alt="Screenshot 2024-03-13 at 22 03 29" src="https://github.com/SeleniumHQ/selenium/assets/33221555/039496c4-b467-4edb-a21e-8b4e5facd3a4">

Reply:
I don't quite follow what populates `.gem_rbs_collection/rubyzip' - I cannot find the rbs_collection.yaml entry for RubyZip. I would probably need to play around with this PR to understand better.

Reply:
The rubyzip signatures are in the .gem_rbs_collection, so when we get all the signatures after doing rbs install it gets all the signatures from the gem_rbs_collection repo

<img width="457" alt="Screenshot 2024-03-16 at 00 33 29" src="https://github.com/SeleniumHQ/selenium/assets/33221555/50d841f6-01f4-4d24-bae1-314d00016300">
<img width="486" alt="Screenshot 2024-03-16 at 00 33 52" src="https://github.com/SeleniumHQ/selenium/assets/33221555/48618466-3864-4c90-bf6b-dabb2fa8e984">